action/README.md

```diff
@@ -0,0 +1,33 @@
+# Functions framework conformance test javascript action
+
+This action runs the Functions Framework conformance tests with the specified
+parameters.
+
+Requires Go to be installed prior to running (e.g. actions/setup-go).
```

Comment:
The action setup runs with Node.js, so that's available by default from the action configuration. However, the action runs a Go binary, and therefore requires Go to be installed. I haven't yet found a way to require a particular action to be run before this action, and thus, this note. I can clarify the version number.